client/starwhale/base/bundle_copy.py

```diff
@@ -277,36 +345,31 @@ def _do_ubd_prepare(
             progress=progress,
             task_id=task_id,
         )
-        upload_id: str = r.json().get("data", {}).get("upload_id", "")
-        if not upload_id:
-            raise Exception("get invalid upload_id")
-        return upload_id
+        return r.json().get("data", {})
 
     def _do_ubd_blobs(
         self,
         progress: Progress,
         workdir: Path,
         upload_id: str,
         url_path: str,
-        add_data_uri_header: bool = False,
+        existed_files: t.List = [],
```

Comment:
`[ ]` as a default value is unsafe.

recommended:

```
def _do_ubd_blobs(
        self,
        progress: Progress,
        workdir: Path,
        upload_id: str,
        url_path: str,
        existed_files: t.Optional[t.List] = None)
     ....
    existed_files = existed_files or []
```
related docs: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
fixed